pkg/cli/expose/scheme.go

```diff
@@ -0,0 +1,38 @@
+package expose
+
+import (
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+
+	"github.com/openshift/api"
+	"github.com/openshift/api/route"
+
+	schemehelper "github.com/openshift/oc/pkg/helpers/scheme"
+)
+
+var (
+	exposeCmdScheme = runtime.NewScheme()
+	exposeCmdCodecs = serializer.NewCodecFactory(exposeCmdScheme)
+)
+
+func init() {
+	utilruntime.Must(api.InstallKube(exposeCmdScheme))
```

Comment:
@ardaguclu as you suggested, I added this here to make sure all the upstream types work as before, but I didn't note any difference with or without it when exposing a service. I'm fine with keeping this if we think some other uses of this command might break without it though.

Reply:
yes, I agree with you